src/test/scala/chiseltest/tests/VecLiteralsSpec.scala

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest.tests
+
+import chisel3._
+import chisel3.experimental.VecLiterals._
+import chiseltest._
+import org.scalatest._
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+
+class VecLiteralsSpec extends AnyFreeSpec with ChiselScalatestTester with Matchers {
```

Comment:
Do you also want to add a Record-in-Vec or vice versa test case?

Reply:
Added
- `Vec` of `Vec`
- `Bundle` of `Vec`
- `Vec` of `Bundle`
- `Bundle` of `Bundle` (this one in BundleLiteralsSpec)